src/Tribe/RSVP.php

```diff
@@ -1314,7 +1314,7 @@ protected function get_attendees_by_order_id( $order_id ) {
 	 *
 	 * @return array
 	 */
-	protected function get_attendees( $attendees_query, $post_id ) {
+	protected function get_attendees( WP_Query $attendees_query, $post_id ) {
```

Comment:
Yeah, I had a discussion about this with @bordoni in another PR: the reason behind it is the abstract has it so it throws warnings (which I see on my local) - figured we should make it match. (Or remove it from the abstract)